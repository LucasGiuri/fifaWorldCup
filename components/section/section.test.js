import React from 'react';

import renderer from 'react-test-renderer';
import 'jest-styled-components';

import Section from './section.tsx';

describe('Image', () => {
  it('basic snapshot renders', () => {
    const b = renderer.create(<Section title='faketitle'>fake children</Section>);
    const tree = b.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
