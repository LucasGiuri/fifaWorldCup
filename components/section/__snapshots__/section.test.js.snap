// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Image basic snapshot renders 1`] = `
.c0 {
  margin: 36px 0;
}

.c1 {
  padding: 16px;
  font-size: 16px;
  font-weight: 600;
  background: lightgray;
  margin-bottom: 16px;
}

<div
  className="c0"
>
  <div
    className="c1"
  >
    faketitle
  </div>
  fake children
</div>
`;
